README.md

```
@@ -2,313 +2,1521 @@
+- [Dev Patel](https://devpatel-freelancer.netlify.app) [.Net Devloper & Freelancer]
+- [Dev Sharma](https://devsh-portfolio8.vercel.app) [Full Stack Developer]
+- [Devansh Baghel](https://baghel.dev) [Full Stack Developer]
+- [Devi Prasad](https://deviprasadkl.github.io/Portfolio/)
+- [Devi](https://devi-r.com)
+- [Devrim Mehmet Pattabanoğlu](https://devrimmehmet.com)
+- [Dewald Els](https://dewaldels.com)
+- [Dhanush Nehru](https://chat-portfolio-dhanushnehru.netlify.app) [Unique Whatsapp Portfolio]
+- [Dhaval Patel](https://dhavalcode.com)
+- [Dheeraj Gupta](https://dheerajgupta.netlify.app/#)
+- [Dhiraj Basavaraju](https://portfolio-dhirajb7.vercel.app)
+- [Dhiraj Bhawsar Dhirajtsx Portfolioo 2.O](https://dhirajbhawsar.in)[Full Stack Web Developer & Freelancer]
```

markdownlint-cli2 (0.18.1)

Reversed link syntax
(https://dhirajbhawsar.in)